src/module/canvas/helpers.ts

```diff
@@ -10,7 +10,15 @@ import { TokenPF2e } from "./token";
 function measureDistanceRect(
```

Comment:
Now rename to `measureDistanceCuboid` :)

Reply:
renamed and references fixed.
Running another test

Reply:
https://user-images.githubusercontent.com/74384067/218332565-e4bfea3a-c2af-4234-96bb-8728bd45875f.MP4